XRTK.Lumin/Packages/com.xrtk.lumin/Runtime/Providers/Controllers/LuminHandControllerDataProvider.cs

```diff
@@ -94,17 +92,20 @@ public override void Update()
 
             if (handTrackingHandle.IsValid)
             {
+                var leftHandData = new HandData();
```

Comment:
Do we really need to create a new one each update?

Reply:
They are structs now, so caching a reference does not make sense anymore.

Reply:
Makes sense?